docs/ref/databases.txt

```diff
@@ -1107,6 +1107,49 @@ Example of a full DSN string::
         "(CONNECT_DATA=(SERVICE_NAME=orclpdb1)))"
     )
 
+.. _oracle-pool:
+
+Connection pool
+---------------
+
+.. versionadded:: 5.1
+
+To use a connection pool with `oracledb`_, you can set the ``"pool"`` in the
```

Comment:
I don't understand this sentence: it is complex.  Perhaps split it up?  Use bullet points?  Cut out text and let the examples show what is needed?